Makefile

```diff
@@ -9,7 +9,7 @@ ENVTEST_VERSION ?= v0.0.0-20240112123317-48d9a7b44e54
 # See https://pkg.go.dev/golang.org/x/tools/cmd/goimports?tab=versions for the last version
 GOIMPORTS_VERSION ?= v0.17.0
 # See https://github.com/onsi/ginkgo/releases for the last version
-GINKGO_VERSION ?= v2.14.0
+GINKGO_VERSION ?= v2.22.0
```

Comment:
is this intentional?

Reply:
Yes, I forgot to add it as part of https://github.com/medik8s/node-maintenance-operator/pull/138/commits/dbb954b904fce3b217effe196e20ddfa835f78db#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R8